assignment/api/src/resources/assignment_es.properties

```diff
@@ -277,7 +277,8 @@ gen.hideduedate=Ocultar fecha de entrega a estudiantes
 grad2=La calificaci\u00f3n introducida, ''{0}'', es mayor que la puntuaci\u00f3n m\u00e1xima permitida, ''{1}''. Realice la correcci\u00f3n o, de lo contrario, vuelva a pulsar el bot\u00f3n para confirmar el valor introducido, ''{0}''.
 
 grad3=Corregida
-
+grade.title=Calificaciones
+grade.delimiter=,
```

Comment:
Never put delimiters into translation files.

Check the formattedText.getDecimalSeparator() method.